src/Vehicle/Vehicle.h

```diff
@@ -694,6 +694,7 @@ class Vehicle : public FactGroup
     Q_PROPERTY(FactGroup* setpoint          READ setpointFactGroup          CONSTANT)
     Q_PROPERTY(FactGroup* estimatorStatus   READ estimatorStatusFactGroup   CONSTANT)
     Q_PROPERTY(FactGroup* terrain           READ terrainFactGroup           CONSTANT)
+    Q_PROPERTY(FactGroup* distanceSensors   READ distanceSensorFactGroup    NOTIFY DistanceSensorReadingsChange) //####
```

Comment:
This is incorrect. As is the additional signal. The distanceSensors point is constant.